lib/ui/tiles/subscribe_button.dart

```diff
@@ -12,7 +13,8 @@ class SubscribeButton extends StatefulWidget {
 
 class _SubscribeButtonState extends State<SubscribeButton> {
   bool marked;
-
+  final _snackBar =
+      SnackBar(content: Text('Web does not support this feature'));
```

Comment:
Update message: Subscribing is not supported on web.
Also add a "Bell" icon along with the message.

Reply:
Done